engine/battle/animations.asm

```diff
@@ -176,8 +176,13 @@ PlayAnimation:
 	ld h, [hl]
 	ld l, a
 .animationLoop
+	vc_hook FPA_006_End
+	vc_blue_hook FPA_007_End
```

Comment:
I wonder if it would be more obvious/explicit to allow `vc_hook` (and `vc_patch/end`) to take an extra arg which would be check for `IF DEF`. Like `vc_hook FPA_006_End, _RED`.